packages/taquito-michelson-encoder/test/tokens/bool.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { BoolToken } from '../../src/tokens/comparable/bool';
+
+describe('Bool token', () => {
+  let token: BoolToken;
+  beforeEach(() => {
+    token = new BoolToken({ prim: 'bool', args: [{ prim: 'int' }], annots: [] }, 0, null as any);
+  });
+
+  describe('EncodeObject', () => {
+    it('Should encode bool properly', () => {
```

Comment:
Consider using distinct names for tests in the same describe block to describe how tests are different.